tests/test_trigger_policy.py

```
@@ -0,0 +1,171 @@
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import pytest
+
+from biblebot.bot import BibleBot
+from biblebot.triggers import (
+    TriggerSource,
+    detect_trigger,
+)
+
+
+def _make_bot(
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_make_bot`

(ANN202)